gp-includes/things/translation.php

```diff
@@ -422,19 +422,51 @@ public function for_translation( $project, $translation_set, $page, $filters = a
 
 		$join_where = implode( ' AND ', $join_where );
 		if ( $join_where ) {
-			$join_where = 'AND '.$join_where;
+			$join_where = 'AND ' . $join_where;
 		}
 
-		$sql_sort = sprintf( $sort_by, $sort_how );
-
+		$fields = 't.*, o.*, t.id as id, o.id as original_id, t.status as translation_status, o.status as original_status, t.date_added as translation_added, o.date_added as original_added';
+		$join = "$join_type JOIN {$wpdb->gp_translations} AS t ON o.id = t.original_id AND t.translation_set_id = " . (int) $translation_set->id;
```

Comment:
No strong opinion here, but I think this is good enough for most cases, and keeps the code simple.